GameFighter/Models/Lancer.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace GameFighter.Models
+{
+    public class Lancer : Warrior
+    {
+        public Lancer()
+        {
+            Attack = 6;
+        }
+
+        public override void Attacks(Warrior warrior, Army warriorsArmy, Army thisArmy = null)
```

Comment:
Why do you use the second army parameter? It's unused for now. I don't see any case so far which can involve the usage of current army

Reply:
In that task I need info about unit after this, to heal while attacking

Reply:
ok, but this functionality will be required only in next task. At this point you shouldn't know about this)

Reply:
Hah, okey then)